contrib/babelfishpg_tsql/sql/upgrades/babelfishpg_tsql--4.3.0--4.4.0.sql

```diff
@@ -10574,6 +10720,13 @@ CREATE OR REPLACE PROCEDURE sys.sp_reset_connection()
 AS 'babelfishpg_tsql', 'sp_reset_connection_internal' LANGUAGE C;
 GRANT EXECUTE ON PROCEDURE sys.sp_reset_connection() TO PUBLIC;
 
+-- Creates database roles in all the existing databases
+CREATE OR REPLACE PROCEDURE sys.babel_create_database_roles()
+LANGUAGE C
+AS 'babelfishpg_tsql', 'create_database_roles_for_all_dbs';
+CALL sys.babel_create_database_roles();
+DROP PROCEDURE sys.babel_create_database_roles();
```

Comment:
This is also being added by Tanzeel, [PR](https://github.com/babelfish-for-postgresql/babelfish_extensions/pull/2970) so one of you have to remove this piece of code.